docs/modules/ROOT/pages/how-to-guides/testing_applications/retriggering_integration_tests.adoc

```diff
@@ -0,0 +1,13 @@
+= Retriggering Integration Tests
+
+Integration test scenarios for a given snapshot can be re-triggered by adding a label to the snapshot.
+
+== Prerequisites
+The snapshot for which tests will be retriggered has completed its initial tests
+
```

Comment:
Add CLI access as a pre-requisite and link to the doc about how to get CLI access with `xref`.

Reply:
I think the one you want is `getting-started/getting_started_in_cli.adoc`.